packages/next/src/routes/rest/buildFormState.ts

```diff
@@ -23,8 +23,32 @@ export const getFieldSchemaMap = (config: SanitizedConfig): FieldSchemaMap => {
 
 export const buildFormState = async ({ req }: { req: PayloadRequest }) => {
   const reqData: BuildFormStateArgs = req.data as BuildFormStateArgs
-
-  // TODO: run ADMIN access control for user
+  const userCollectionSlug = req.user?.collection
```

Comment:
Let's rename this to `incomingUserSlug`
```suggestion
  const incomingUserSlug = req.user?.collection
```